server/meta.jsx

```diff
@@ -95,7 +107,11 @@ function isLocalUrl(host : string) : boolean {
 }
 
 function validateSDKUrl(sdkUrl : string) {
-    const { protocol, host, hostname, pathname, query, hash } = urlLib.parse(sdkUrl, true);
+    const processedUrl = sdkUrl.replace(/merchant-id=([^\&$]+)/, (_val, match) => {
```

Comment:
I'm really confused about what is happening here. It is the merchant's job to correctly encode the url, why are we doing it here?